libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1848,7 +1848,7 @@ bool TypeChecker::visit(FunctionCall const& _functionCall)
 		if (functionType->takesArbitraryParameters())
 			m_errorReporter.typeError(
 				_functionCall.location(),
-				"Named arguments cannnot be used for functions that take arbitrary parameters."
+				"Named arguments cannot be used for functions that take arbitrary parameters."
```

Comment:
Is there no test case for this? Also please pull this into a separate PR (can be shared with the other I proposed just above).

Reply:
Will be pulled out together with the one above.